code-challenges.js

```diff
@@ -6,63 +6,133 @@
 
 // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in two strings and determines which of the two strings has more characters. Use the two sets of test variables provided.
 
-// Pseudo code:
-
-// Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
-// Expected outcome: "banana"
+// Pseudo code:  
+// Create declarations and parameters 
+// Create if statement, using .length and greater symbol that will compare the two strings
+// Create else if statement, using .length and greater symbol that will compare the two strings if the opposite were true
+// Run the function using console.log to verify proper returns
+
+// const fruit1 = "apple"
+// const fruit2 = "banana"
+
+// const longerFood = (fruit1, fruit2) => {
+//     if(fruit1.length > fruit2.length)
+//         return `${fruit1}`
+//     else if(fruit2.length > fruit1.length)
+//         return `${fruit2}`
+    
+//     }
+// console.log(longerFood(fruit1 , fruit2))
+
+// // // Expected outcome: "banana"
+
+// const fruit3 = "cherry"
+// const fruit4 = "kiwi"
+
+// const foodLonger = (fruit3, fruit4) => {
+//     if(fruit3.length > fruit4.length)
+//         return `${fruit3}`
+//     else if(fruit4.length > fruit3.length)
+//         return `${fruit4}`
+// }
+// console.log(foodLonger(fruit3, fruit4))
 
-// Set two:
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
 // Expected outcome: "cherry"
 
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Pseudo code:
-
-const temp1 = 42
+// Create declarations and parameters, where temp is the number as it relates to 212 degrees F
+// Create if, else if statements along with greater/lesser/strictly equal symbols to determine below/at/boiling temps based on variables given
+// Run the function using console.log to verify accuracy of function
```

Comment:
Very nice pseudocode! If you wanted to be even more descriptive, you could mention string interpolation and start thinking about how you could incorporate an else.